### PR TITLE
React portals (again)

### PR DESCRIPTION
**Pull request by Fer0x**
Picks up work from https://github.com/ryanseddon/react-frame-component/pull/93

Fix rendering in Firefox and get rid of `setTimeout`.

### CODE REVIEW

File: src/Frame.jsx [hunk 1]
```diff
@@ -57,24 +39,23 @@ export default class Frame extends Component {
...
-    this.renderFrameContents();
-  }
 
-  componentDidUpdate() {
-    this.renderFrameContents();
+    const doc = this.getDoc();
+    if (doc && doc.readyState === 'complete') {
+      this.forceUpdate();
+    } else {
+      this.node.addEventListener('load', this.handleLoad);
```
Comment (ryanseddon) [inline src/Frame.jsx]: Oh clever

File: src/Frame.jsx [hunk 2]
```diff
@@ -85,47 +66,51 @@ export default class Frame extends Component {
...
-      resetWarnings();
-    } else {
-      setTimeout(this.renderFrameContents.bind(this), 0);
-    }
+    return (
+      <div>
+        {ReactDOM.createPortal(this.props.head, this.getDoc().head)}
+        {ReactDOM.createPortal(contents, mountTarget)}
+      </div>
+    );
```
Comment (ryanseddon) [inline src/Frame.jsx]: You can drop the div here and just return an array:

```js
return [
  ReactDOM.createPortal(this.props.head, this.getDoc().head),
  ReactDOM.createPortal(contents, mountTarget)
];
```

File: src/Content.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, { Component, Children } from 'react'; // eslint-disable-line no-unused-vars
+import PropTypes from 'prop-types';
+
+export default class Content extends Component {
```
Comment (ryanseddon) [inline src/Content.jsx]: Could you add a test for this component too please

Comment (Fer0x) [reply src/Content.jsx]: @ryanseddon done

**Pull request merged**